pyrdfj2/sparqlbuilder.py

```diff
@@ -0,0 +1,72 @@
+import logging
+from pathlib import Path
+
+from jinja2 import Environment, FileSystemLoader, meta
+
+from pyrdfj2.j2_functions import Filters, Functions
+from pyrdfj2.pyrdfj2 import SparqlBuilder
+
+log = logging.getLogger(__name__)
+
+
+DEFAULT_TEMPLATES_FOLDER = Path(__file__).parent.absolute() / "templates"
```

Comment:
the contents of this folder is tuned to sparql -- which is the focus of (only) `pykg2tbl`

feels overreach to push it up here ?
from the other side: we should not put the `pysubyt` templates up here either? (allthough those are less likely to be meaningful) 

note -- you might want to move (some sparql and some ttl?) templates into the tests/ though in order to verify and asses

Reply:
not quite sure want you mean here.

I though we were making these templates into the package default ones. But I can leave it without and then its only up to the user (pykg2tbl) to have templates.
This would also change the testing here.


About `pysubyt`. 
I struggled to decide what to migrate from `pysubyt` to here, and I am still not sure If i migrated enough or not.
I believe I brought enough code to clean `pysubyt` a bit, but any more we would need to refactor it. (That was my feeling)
If we want to add some templates from there, we might need to get more code (and thus refactoring such code)